Doc - Add missing Airflow properties and update docker-compose (#125)
Avaiga/taipy-doc: docs/manuals/core/config/job-config.md

    airflow_api_retry = "10:int"
    ```

> Airflow will try to check if the task is ready to be executed using PythonSensor, which will trigger every 20 seconds by default. You can update the number of seconds between each check by:

Reviewer: ```suggestion
Before executing a task, Airflow checks if its inputs are ready every 20 seconds by default. You can update the number of seconds between each check by:
```
